### PR TITLE
Update api doc for enabling TcpStore on Windows

### PR DESCRIPTION
**Pull request by gunandrose4u**
Fixes #{issue number}

### CODE REVIEW

File: docs/source/distributed.rst [hunk 1]
```diff
@@ -59,15 +59,16 @@ included if you build PyTorch from source. (e.g.building PyTorch on a host that
 installed.)
 
 .. warning ::
```
Comment (malfet) [inline docs/source/distributed.rst:61]: Should we change warning to note here, as we have a backend parity between Windows and Linux?

File: docs/source/distributed.rst [hunk 2]
```diff
@@ -59,15 +59,16 @@ included if you build PyTorch from source. (e.g.building PyTorch on a host that
...
-    As of PyTorch v1.7, Windows support for the distributed package only covers collective
-    communications with Gloo backend, `FileStore`, and `DistributedDataParallel`. Therefore,
-    the `init_method` argument in :func:`init_process_group` must point to a file. This works
-    for both local and shared file systems:
+    As of PyTorch v1.8, Windows support for the distributed package covers collective
+    communications with Gloo backend, `FileStore`, `Tcp Store`and `DistributedDataParallel`.
+    Therefore, the `init_method` argument in :func:`init_process_group` can point to a file.
+    This works for both local and shared file systems:
```
Comment (malfet) [inline docs/source/distributed.rst:65]:
```suggestion
    As of PyTorch v1.8, Windows supports all collective communications backend but NCCL,
    If  the `init_method` argument of :func:`init_process_group` points to a file it must adhere
    to the following schema:
```

**Review by malfet — APPROVED** [review]
LGTM, but please consider rewording the warning. Also, since you are making small cleanups to the document, please add space between `e.g.` and `building` on line 58, i.e. `e.g. building PyTorch`)

File: docs/source/distributed.rst (hunk 1 above)
Comment (gunandrose4u) [reply docs/source/distributed.rst]: Sure, just resolved.

**Pull request closed without merging**